File: fizzbuzz.py
def fizzbuzz(n: int):
    if n % 15 == 0:
        print("FizzBuzz")
    elif n % 3 == 0:
        print("Fizz")
    elif n % 5 == 0:
        print("Buzz")
    else:
        print(n)

def main():
    for i in range(1, 101):
        fizzbuzz(i);

if __name__ == "__main__":
    main()

File: fizzbuzz.js
function fizzbuzz(n) {
    if (n % 15 === 0) console.log("FizzBuzz");
    else if (n % 3 === 0) console.log("Fizz");
    else if (n % 5 === 0) console.log("Buzz");
    else console.log(n);
}

for (let i = 1; i <= 100; i++) {
    fizzbuzz(i);
}
